pkg/gen/input/workflows/internal/file/create_release_pr.yaml.template

```diff
@@ -45,10 +49,17 @@ jobs:
         run: |
           head="${{ inputs.branch || github.event.ref }}"
           echo "::set-output name=branch::${head}"
+
           head="${head#refs/heads/}" # Strip "refs/heads/" prefix.
-          base="$(echo $head | cut -d '#' -f 1)"
+          if [[ $(echo "$head" | grep -o '#' | wc -l) -gt 1 ]]; then
+            base="$(echo $head | cut -d '#' -f 1)"
+          else
+            base="${{ github.event.base_ref }}"
+          fi
+
           base="${base#refs/heads/}" # Strip "refs/heads/" prefix.
-          version="$(echo $head | cut -d '#' -f 3)"
+
+          version="$(echo $head | rev | cut -d '#' -f 1 | rev)"
```

Comment:
Basically taking the last portion of the `master#release#minor` string (instead of the 3rd portion), because this string can now also be `release#minor`.

```
$ rev --help
Usage: rev [options] [file ...]

Reverse lines characterwise.
```

Reply:
This feels a little awkward. How about `version="$(echo $head | awk -F# '{print $NF}')"`? `-F` is field separator, `$NF` is number of fields (aka get the last field separated by `#`).
```
> echo 'release#v1.0.0' | awk -F# '{print $NF}' 
v1.0.0
> echo 'master#release#v1.0.0' | awk -F# '{print $NF}'
v1.0.0
```

Reply:
My lack of `awk` is disturbing ... but thank for the suggestion <3

Reply:
My lack of `awk` is disturbing ... but thank for the suggestion <3